libraries/ms-common/src/stm32f0xx/spi.c

```diff
@@ -49,17 +47,15 @@ StatusCode spi_init(SPIPort spi, SPISettings *settings) {
   gpio_settings.alt_function = GPIO_ALTFN_NONE;
   gpio_init_pin(&settings->cs, &gpio_settings);
 
-  SPI_InitTypeDef init = {
-    .SPI_Direction = SPI_Direction_2Lines_FullDuplex,
-    .SPI_Mode = SPI_Mode_Master,
-    .SPI_DataSize = SPI_DataSize_8b,
-    .SPI_CPHA = (settings->mode & 0x01) ? SPI_CPHA_2Edge : SPI_CPHA_1Edge,
-    .SPI_CPOL = (settings->mode & 0x02) ? SPI_CPOL_High : SPI_CPOL_Low,
-    .SPI_NSS = SPI_NSS_Soft,
-    .SPI_BaudRatePrescaler = (index - 2) << 3,
-    .SPI_FirstBit = SPI_FirstBit_MSB,
-    .SPI_CRCPolynomial = 7
-  };
+  SPI_InitTypeDef init = { .SPI_Direction = SPI_Direction_2Lines_FullDuplex,
```

Comment:
Yeah, I reverted it but if we are going to move to support clang-format it might change back?